### ISSUE TITLE
False negative for E12

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#108]
I think we should get one of the E12 errors on the following input.

``` python


def long_function_name(
    var_one, var_two, var_three,
        var_four):
    print(var_one)
```
